proposals/low-level-struct-improvements.md

```diff
@@ -351,12 +351,25 @@ Span<int> ComplexScopedRefExample(scoped ref Span<int> span)
 
 <a name="rules-method-arguments-must-match"></a>
 
-The presence of `scoped` allows us to also refine the method arguments must match rule. It can now be reduced to 
+The presence of `ref` fields means the rules around method arguments must match need to be updated as a `ref` parameter can now be stored as fields in a `ref struct` argument to the method. Previously the rule only had to consider other `ref struct` being stored as fields. The impact of this is discussed in [the compat considerations](#compat-considerations). 
+
+The presence of `scoped` allows us to refine the rule to reduce the friction the rule creates. The `scoped` modifier lets us remove arguments from consideration as they cannot be returned from the method.
+
+The updated rule is:
 
-> For a method invocation if there is an argument for a `ref` or `scoped ref` parameter of a `ref struct` type where the argument has *safe-to-escape* E1 then no argument may contribute a narrower *safe-to-escape* than E1.
+> For a method invocation if there is an argument for a `ref`, `scoped ref` or `out` parameter of a `ref struct` type where the argument has *safe-to-escape* E1 then 
+> 1. No `ref struct` argument may contribute a narrower *safe-to-escape* than E1.
+> 2. No `ref` or `in` argument may contribute a narrower *ref-safe-to-escape* than E1.
+>
+> This rule does not apply to `ref struct` arguments for `in` parameters
 
 Impact of this change is discussed more deeply [below](#examples-method-arguments-must-match). Overall this will allow developers to make call sites more flexible by annotating non-escaping ref-like values with `scoped`.
 
+The `scoped` modifier on parameters also impacts our OHI and `delegate` conversion rules. The signature for an override, interface implementation or `delegate` conversion can: 
```

Comment:
>OHI [](http://example.com/codeflow?start=53&length=3)

Consider using "overriding and interface implementation" instead. It would avoid introducing a new "OHI" term, and also `scoped` does not affect hiding.

Reply:
Tried to clear this up.